content/blog/2019-08-01-new-react-devtools.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Introducing the new React DevTools"
```

Comment:
Nit: capitalize "New"? We don't always do that but mostly do for announcement posts

Reply:
That's what threw me off. I couldn't tell if these were new or a new version (hence my comment about "a lot has changed since..." above!)